src/sentry/static/sentry/app/components/missingProjectMembership.jsx

```diff
@@ -25,28 +26,25 @@ const MissingProjectMembership = React.createClass({
       loading: true,
     });
 
-    this.api.joinTeam(
-      {
-        orgId: this.props.organization.slug,
-        teamId: this.props.team.slug,
+    joinTeam(this.api, {
+      orgId: this.props.organization.slug,
+      teamId: this.props.team.slug,
+    }).then(
```

Comment:
its too bad we lose the named success and error here, those kind of help readability imo. maybe `catch` and `then` would be better? (shrug)